### PR TITLE
add libebur128 package

### PR DESCRIPTION
**Pull request by andrewrk**
This is my first nix package, please review carefully.

### CODE REVIEW

File: pkgs/development/libraries/libebur128/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, cmake, speexdsp, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "libebur128-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/jiixyj/libebur128.git";
+    rev = "refs/tags/v1.0.2";
```
Comment (pSub) [inline pkgs/development/libraries/libebur128/default.nix]: You can use string interpolation to insert the version number, i.e. `rev = "refs/tags/${version}";`

File: pkgs/development/libraries/libebur128/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, cmake, speexdsp, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "libebur128-${version}";
+
+  src = fetchgit {
```
Comment (pSub) [inline pkgs/development/libraries/libebur128/default.nix]: Please use `fetchFromGitHub`, that avoids cloning the whole repo.

**Comment by pSub** [comment]
Looks good to me otherwise.

**Comment by andrewrk** [comment]
Updated, and ready to go.

**Comment by pSub** [comment]
Thanks!

**Pull request merged**